### ISSUE TITLE
New Feature : Search / File Filter

### ISSUE THREAD

**Issue by supportreq** [ladrua/Open-ownCloud#1]
Adding a search bar at the top to allow users to filter the file list.

**Comment by ladrua** [comment]
Are you talking about a search filter(sort by name, type...), or text search? I guess both would be nice, to add a view filter is what I will get on first.

**Comment by supportreq** [comment]
No plain text search.. Similar to what dropbox has to offer.. if i enter abc, it will give me files and folders with abc.. this would be an amazing feature..

**Comment by ladrua** [comment]
I see, yes, would be a nice feature, its a bit of work tho. how its set up now, it only queries the path thats viewed. So it only has the files/folders of the current view in its array. This could be fixed of course, but is a bit of work.

**Comment by supportreq** [comment]
can we take only the file name such that the search works and rest operations remain the same?

**Comment by ladrua** [comment]
what do you mean? That it only searches the files and not folders? It wouldnt make a difference. Same work would need to be done.

**Comment by supportreq** [comment]
@ladrua yeah i realized it when I pressed the comment button.. let me know in case you need any help for testing the code..

i can help with ipad testing also..

**Comment by ladrua** [comment]
@supportreq thanks, there is still a lot of work before a release, and will prioritise iPhone.

**Comment by supportreq** [comment]
Sure I will help also in iPhone.. Let me know
